### ISSUE TITLE
the error of loss function

### ISSUE THREAD

**Issue by liujiang137** [ronghuaiyang/arcface-pytorch#25]
The value of the loss function cannot be lowered，When I execute to the 50th epoch, this value is still 12.137....，It is similar to the previous epoch.
I listed the process of two epoch
![image](https://user-images.githubusercontent.com/45035204/55682511-4a9fc900-5967-11e9-836e-078099996495.png)
![image](https://user-images.githubusercontent.com/45035204/55682525-6f943c00-5967-11e9-8ef4-18f85fe1e51f.png)

**Comment by luluvi** [comment]
> The value of the loss function cannot be lowered，When I execute to the 50th epoch, this value is still 12.137....，It is similar to the previous epoch.
> I listed the process of two epoch
> ![image](https://user-images.githubusercontent.com/45035204/55682511-4a9fc900-5967-11e9-836e-078099996495.png)
> ![image](https://user-images.githubusercontent.com/45035204/55682525-6f943c00-5967-11e9-8ef4-18f85fe1e51f.png)

How do you get the data set training, can you give me some ideas?
thank you!

**Comment by hfeng-xia** [comment]
Hi, I have met the same problem. How do you solve this problem?

**Comment by Woolseyyy** [comment]
I check the loss function(cocal_loss). It seems to be different from the theory,
